### ISSUE TITLE
Onedrive: Null Exception

### ISSUE THREAD

**Issue by svensvond** [Kyrodan/KeeAnywhere#389]
Suddently (17-12-2023) this appeared, even if i re-add my onedrive account(s):

![image](https://github.com/Kyrodan/KeeAnywhere/assets/37634123/d2c6235e-d2f7-4ed0-8fa5-62066d844f92)

**Comment by bcrosser** [comment]
![image](https://github.com/Kyrodan/KeeAnywhere/assets/69265/54f1ffe2-bb2a-45f6-b110-2c4500bdc12e)
I get this message when I try to open a file from OneDrive, and every time I try to open a keypass store it gives me an error and tries to reauthorize me.

**Comment by StraggleCraft** [comment]
try this work around:
Start KeePass as an administrator
Reauthorize
Exit,
Restart KeePass as regular user

**Comment by lostgit** [comment]
See the solution I reported on [https://github.com/Kyrodan/KeeAnywhere/issues/282](url). 

TL;DR remove the existing OneDrive credential from Tools -> KeeAnywhere and recreate it.
